docs/guide/front-end-assets.md

```diff
@@ -127,3 +127,44 @@ Here's how it works in a typical project with its own build process:
 - Whenever code changes are made, `nodemon` automatically restarts this cycle and refreshes the browser after a successful restart.
 
 But you don't have to use SCSS, webpack or any other specifics mentioned here, except for pushing the output into a `ui/public` folder so that Apostrophe can find it. You can follow whatever process works best for you or your organization.
+
+### The `public` folder
+
+The `public` folder of each module solves a different problem from `ui/public`. While `ui/public` is for JavaScript and CSS files that should be appended as-is to Apostrophe's JavaScript and CSS bundles, usually to accommodate a custom webpack build, `public` is for files that should be **available separately.** A common example is a `.png` file to be used as a background image in CSS.
+
+While you can place asset files in the `public` folder of the project itself, and reference them with URLs like `/images/bg.png`, this has two problems when used in stylesheets:
```

Comment:
To be clear about which folder, how about:
> the root `/public` folder

Reply:
We're using the first person plural in these guides, so here it'd be "While we can place asset files...". There are some others below as well.

Reply:
In some cases it could be avoided altogether, like "`/images/bg.png` in CSS code" (no pronoun).